Meta/WPT.sh

```diff
@@ -272,22 +547,30 @@ compare_wpt() {
     ensure_wpt_repository
     METADATA_DIR=$(mktemp -d)
     pushd "${WPT_SOURCE_DIR}" > /dev/null
-      ./wpt update-expectations --product ladybird --full --metadata="${METADATA_DIR}" "${INPUT_LOG_NAME}"
+        ./wpt update-expectations --product ladybird --full --metadata="${METADATA_DIR}" "${INPUT_LOG_NAME}"
     popd > /dev/null
     WPT_ARGS+=( "--metadata=${METADATA_DIR}" )
     build_ladybird_and_webdriver
     execute_wpt "${@}"
     rm -rf "${METADATA_DIR}"
 }
 
-if [[ "$CMD" =~ ^(update|run|serve|compare|import|list-tests)$ ]]; then
+if [[ "$CMD" =~ ^(update|clean|run|serve|compare|import|list-tests|testx)$ ]]; then
```

Comment:
Could we add the new commands (`clean`, `testx`) to the `print_help` output above?

It's not immediately clear by the name what the `testx` command does, or what the use case is. Not sure what a better name would be though.

Reply:
ah `testx` is just a test thing, I'll remove it before merge.
Will add `clean` to the help though, thanks